hera_librarian/store.py

```diff
@@ -134,7 +134,7 @@ def copy_to_store(self, local_path, store_path):
         argv = [
             'rsync',
             '-aP',
-            '-e', 'ssh -c arcfour256 -o BatchMode=yes -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no',
+            '-e', 'ssh -o BatchMode=yes -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no',
```

Comment:
You probably needed this change for local testing? That speaks to a broader issue (we use arcfour onsite for speed, but it's insecure and so disabled out of the box), but it shouldn't be changed in this pull request.